xml/entity-decl.ent

```diff
@@ -197,9 +197,9 @@
 <!ENTITY productnamereg "<phrase xmlns='http://docbook.org/ns/docbook' role='productname'><phrase os='osuse'>&opensusereg; Leap</phrase><phrase os='sles'>&slsreg;</phrase><phrase os='sled'>&sledreg;</phrase><phrase os='slemicro'>&slemreg;</phrase></phrase>">
 <!ENTITY productnameshort "<phrase xmlns='http://docbook.org/ns/docbook' role='productname'><phrase os='osuse'>&opensuse; Leap</phrase><phrase os='sles'>&slsa;</phrase><phrase os='sled'>&sleda;</phrase><phrase os='slemicro'>&slema;</phrase></phrase>">
 <!ENTITY product-ga    "15">
-<!ENTITY product-sp    "3">
+<!ENTITY product-sp    "4">
 <!ENTITY productnumbershort "&product-ga;.&product-sp;">
```

Comment:
This is an entirely very ungood entity name. It's not the official short variant of the product version. Instead, it is only a URL component for some SLE registration and Leap repo URLs. To make use cases clearer, I'd suggest replacing with two distinct entities:

```suggestion
<!ENTITY productnumber-regurl "&product-ga;.&product-sp;">
<!ENTITY productnumber-leaprepo "&product-ga;.&product-sp;">
```

There will be fallout from changing this entity name, which must be fixed separately:

```make
[replace with productnumber-regurl]   xml/containers-docker-installation.xml:74:<screen>&prompt.sudo;SUSEConnect -p sle-module-containers/&productnumbershort;/x86_64</screen>
[replace with productnumber-regurl]   xml/klp.xml:232:Activate with: SUSEConnect -p sle-module-live-patching/&productnumbershort;/x86_64 \
[replace with productnumber-regurl]   xml/klp.xml:242:<screen>SUSEConnect -p sle-module-live-patching/&productnumbershort;/x86_64 \
[replace with productnumber-leaprepo] xml/rmt_migrate_from_smt.xml:816:<screen>http://download.opensuse.org/debug/distribution/leap/&productnumbershort;/repo/oss </screen>
[replace with productnumber-leaprepo] xml/rmt_migrate_from_smt.xml:820:<screen>/usr/share/rmt/public/repo/debug/distribution/leap/&productnumbershort;/repo/oss</screen>
[replace with productnumber]          xml/security_ldap.xml:15:    management, and address management. In &productname; &productnumbershort; the
[replace with productnumber]          xml/security_xca.xml:19:        prefer a graphical tool, &sle; &productnumbershort; includes XCA,
```

Reply:
@taroth21 asked that SLES and SLED be changed, I expect Jana is responsible for SLE Micro. See doku-intern 'SLE 15 SP4: Updating the needed infrastructure Wed, 3 Nov 2021'

Reply:
I agree that your suggested entity name is an improvement-- shouldn't all SLE 15 releases have this change?

Reply:
see https://github.com/SUSE/doc-sle/pull/1001 for MAIN file changes

Reply:
eh too much overthinking re sle micro...